src/NuGet.Core/NuGet.PackageManagement/BuildIntegration/DependencyGraphRestoreUtility.cs

```diff
@@ -238,26 +238,29 @@ public static async Task<PackageSpec> GetProjectSpec(IDependencyGraphProject pro
                         packageSpec.RestoreMetadata.ProjectStyle == ProjectStyle.Standalone) // Don't add global tools to restore specs for solutions
                     {
                         dgSpec.AddRestore(packageSpec.RestoreMetadata.ProjectUniqueName);
-                    }
-
-                    var projFileName = Path.GetFileName(packageSpec.RestoreMetadata.ProjectPath);
-                    var dgFileName = DependencyGraphSpec.GetDGSpecFileName(projFileName);
-                    var persistedDGSpecPath = Path.Combine(packageSpec.RestoreMetadata.OutputPath, dgFileName);
 
-                    if (File.Exists(persistedDGSpecPath))
-                    {
-                        var persistedDGSpec = DependencyGraphSpec.Load(persistedDGSpecPath);
+                        var projFileName = Path.GetFileName(packageSpec.RestoreMetadata.ProjectPath);
+                        var dgFileName = DependencyGraphSpec.GetDGSpecFileName(projFileName);
+                        var outputPath = packageSpec.RestoreMetadata.ProjectStyle == ProjectStyle.ProjectJson ?
```

Comment:
Wouldn't this mean that you are looking for the dg in the same directory as the project.json file? 
Why isn't it written in the obj folder similar to the cache file?

Reply:
yes it would... honestly I don't care either way... I just did it to make some tests happy since `Output` path wasn't being set in there which won't happen in real scenarios. So I'll just update the tests without changing the code.